scripts/backupmanager.sh

```diff
@@ -0,0 +1,336 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+source /includes/colors.sh
+
+# Default values if the environment variables exist
+
+BACKUP_DIR=${BACKUP_PATH} # Dir where the backup files are stored
+# RESTORE_FILE_PATH=${TRIGGER_RESTORE_PATH} # Dir where the restore trigger file with the backup file name is stored
+# BACKUP_FILE=${TRIGGER_RESTORE_FILE} # Path of the backup file trigger
+SAVE_DIR=${GAME_SAVE_PATH} # Dir where the game save files are stored
+BACKUPS_TO_KEEP=${BACKUP_RETENTION_AMOUNT} # Number of backup files to keep
+
+
+# Function to print usage information
+function print_usage() {
+    script_name=$(basename "$0")
+    echo "Usage:"
+    echo "  ${script_name} [backup_directory] -c|--create"
+    # echo "  ${script_name} [backup_directory] -r|--restore [backup_name]"
```

Comment:
No commented update code in production, if its done, put it in, if not, then remove it.

Only # comments for context and links and sources.